linters/bandit.js

```diff
@@ -53,7 +53,16 @@ module.exports = class Bandit {
    * @param {Buffer} data The raw linter results data
    */
   parseResults (data) {
-    return JSON.parse(data)
+    let parsedData = JSON.parse(data)
+    if (parsedData.errors.length !== 0) {
+      let errStr = '\n'
+      for (let i = 0; i < parsedData.errors.length; ++i) {
+        errStr += 'issue ' + (i + 1) + ': ' + parsedData.errors[i].reason + ': ' +
```

Comment:
It looks odd to start a message in lower case.

Reply:
Well, the full message looks like the one from the picture.

Reply:
The PR message still states "issue 1...."